docs/content/preview/api/ysql/exprs/geo_partitioning_helper_functions/func_yb_server_region.md

```diff
@@ -44,20 +44,20 @@ Do the following to create a 3-node multi-region cluster and a geo-partitioned t
     ```sh
     ./bin/yugabyted start                           \
       --base_dir=/home/yugabyte/<IP1>/yugabyte-data \
-      --listen=<IP1>                                \
```

Comment:
If we are making changes to replace `--listen` to `--advertise_address`. We should also go ahead and replace this 

```
--master_flags "placement_cloud=aws,placement_region=us-east-2,placement_zone=us-east-2c" \
      --tserver_flags "placement_cloud=aws,placement_region=us-east-2,placement_zone=us-east-2c"
```

with 
```

 ./bin/yugabyted start                           \
      --base_dir=/home/yugabyte/<IP2>/yugabyte-data \
      --advertise_address=<IP2>                     \
      --cloud_location=aws.us-east-2.us-east-2c     \
      --join=<IP1>
 ```

Reply:
@sudhanshu456 can you make these changes

Reply:
Yes, will do

Reply:
Made the changes please take a look!